### PR TITLE
Add missing affiliations for authors

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
There is an issue in [generating PDF](https://github.com/openjournals/joss-reviews/issues/7171#issuecomment-2327379155) of the paper during review because affiliations are missing for some authors.

There is still one missing affiliation that I can't find.

### CODE REVIEW

File: paper/paper.Rmd [hunk 1]
```diff
@@ -7,16 +7,16 @@ authors:
...
+    orcid: 0000-0002-8999-0738
   - name: Michael Chirico
     affiliation: 2
     orcid: 0000-0003-0787-087X
   - name: Russ Hyde
     affiliation: 5
     orcid: ~
   - name: Ren Kun
-    affiliation: ~
+    affiliation: 7
```
Comment (IndrajeetPatil) [inline paper/paper.Rmd:19]: @renkun-ken Can you please let us know what's your current affiliation? I couldn't figure it out from [your website](https://renkun.me/about/).

File: paper/paper.Rmd [hunk 2]
```diff
@@ -35,6 +35,10 @@ affiliations:
     name: Preisenergie GmbH, Munich, Germany
   - index: 5
     name: Jumping Rivers
+  - index: 6
+    name: The University of Queensland
+  - index: 7
+    name: Unknown
```
Comment (IndrajeetPatil) [inline paper/paper.Rmd:41]: Temporary entry to make sure that PDF can be built: https://github.com/openjournals/joss-reviews/issues/7171#issuecomment-2327379155

**Review by MichaelChirico — APPROVED** [review]

File: paper/paper.Rmd (hunk 1 above)
Comment (MichaelChirico) [reply paper/paper.Rmd:19]: recommend emailing directly to ask as well

Comment (IndrajeetPatil) [reply paper/paper.Rmd:19]: Good idea!

Comment (IndrajeetPatil) [reply paper/paper.Rmd:19]: I had written to Ren Kun, but still no response to my e-mail even after 2 weeks. I am not sure how to proceed here. 

Should we retain “Unknown” as the affiliation?

Comment (MichaelChirico) [reply paper/paper.Rmd:19]: It seems reasonable to me.

**Pull request merged**